ci/run_cloudops_tests_osp18.yml

```diff
@@ -13,10 +13,12 @@
         - name: Include vars from the extra_vars files
           ansible.builtin.include_vars:
             dir: "{{ cifmw_basedir }}/artifacts/parameters"
+          when: hostvars['localhost'].enable_autoscaling_role | bool
```

Comment:
Guarding the block is sufficient here. The individual task don't need to be guarded.